e2e-tests/playwright/utils/UIhelper.ts

```diff
@@ -147,34 +141,44 @@ export class UIhelper {
   }
 
   async verifyRowsInTable(
-    rowTexts: string[] | RegExp[],
+    rowTexts: (string | RegExp)[],
     exact: boolean = true,
   ) {
     for (const rowText of rowTexts) {
-      const rowLocator = this.page
-        .locator(`tr>td`)
-        .getByText(rowText, { exact: exact })
-        .first();
+      await this.verifyTextInLocator(`tr>td`, rowText, exact);
+    }
+  }
+
+  async verifyText(text: string | RegExp, exact: boolean = true) {
+    await this.verifyTextInLocator('', text, exact);
+  }
+
+  private async verifyTextInLocator(
+    locator: string,
+    text: string | RegExp,
+    exact: boolean,
+  ) {
+    const elementLocator = locator
+      ? this.page.locator(locator).getByText(text, { exact }).first()
+      : this.page.getByText(text, { exact }).first();
+
+    try {
+      await elementLocator.waitFor({ state: 'visible', timeout: 10000 });
+      await elementLocator.waitFor({ state: 'attached', timeout: 10000 });
 
       try {
-        await rowLocator.waitFor({ state: 'visible', timeout: 10000 });
-        await rowLocator.waitFor({ state: 'attached', timeout: 10000 });
-
-        try {
-          await rowLocator.scrollIntoViewIfNeeded();
-        } catch (error) {
-          console.warn(
-            `Warning: Could not scroll element into view. Error: ${error.message}`,
-          );
-        }
-
-        await expect(rowLocator).toBeVisible();
+        await elementLocator.scrollIntoViewIfNeeded();
       } catch (error) {
-        console.error(
-          `Error: Failed to verify row with text "${rowText}". Error: ${error.message}`,
+        console.warn(
+          `Warning: Could not scroll element into view. Error: ${error.message}`,
         );
       }
+    } catch (error) {
```

Comment:
Can we remove the outer try-catch? It doesn't seem to serve any purpose now since you moved the assertion out of it.